<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QDialog" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/Resources/logInBackground.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0d0c0c;
border-radius: 40px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>EDIT ACCOUNT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="budgetSpinBox">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid gray;
    border-radius: 10px;
    padding-right: 15px; /* Zostaw miejsce na przyciski góra/dół */
    padding-left: 5px;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string> zł</string>
   </property>
   <property name="minimum">
    <double>-100000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="confrimButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">button-&gt;setStyleSheet(
    &quot;QPushButton {&quot;
    &quot;   background-color: white;&quot;
    &quot;   color: black;&quot;
    &quot;   border-radius: 10px;&quot;
    &quot;   border: none;&quot;
    &quot;   padding: 10px;&quot;
    &quot;}&quot;
    &quot;QPushButton:hover {&quot;
    &quot;   background-color: rgba(255, 255, 255, 0.9);&quot;
    &quot;}&quot;
    &quot;QPushButton:pressed {&quot;
    &quot;   background-color: rgba(255, 255, 255, 0.8);&quot; // Lekkie przyciemnienie po naciśnięciu
    &quot;}&quot;
);</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>REPEAT PASSWORD</string>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>blackLabel</zorder>
  <zorder>titleLabel</zorder>
  <zorder>passwordLineEdit1</zorder>
  <zorder>emailLineEdit</zorder>
  <zorder>budgetSpinBox</zorder>
  <zorder>confrimButton</zorder>
  <zorder>passwordLineEdit2</zorder>
  <zorder>usernameLineEdit</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
